openapi.yaml

```
@@ -3727,7 +3747,7 @@ paths:
             application/json:
               schema:
                 properties:
-                  '': { type: array, items: { type: object } }
+                  repositories: { type: array, items: { type: object } }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)